src/core/internal/elf/dl.d

```diff
@@ -74,9 +74,10 @@ struct SharedObject
      */
     static bool findForAddress(const scope void* address, out SharedObject result)
     {
-        version (linux)       enum IterateManually = true;
-        else version (NetBSD) enum IterateManually = true;
-        else                  enum IterateManually = false;
+        version (linux)        enum IterateManually = true;
+        else version (NetBSD)  enum IterateManually = true;
+        else version (Solaris) enum IterateManually = true;
+        else                   enum IterateManually = false;
```

Comment:
This module is currently disabled for Solaris (`version (LinuxOrBSD)`) - enable?

Reply:
I hadn't considered that. :-)